### PR TITLE
[PR] Fixing timer paths and stop function

### PR DESCRIPTION
**Pull request by LuchoTurtle**
related to #256

This addresses the issues brought up in https://github.com/dwyl/mvp/issues/256#issuecomment-1384091996 and https://github.com/dwyl/mvp/issues/256#issuecomment-1384104504.

This PR adds the `stop` endpoint in `api/timers/:id/stop` that accepts *no attributes* (so it's not redundant with the pre-existent `update` function).
When creating a `timer`, it now has a default `start` value if none is passed.

This PR branches off from https://github.com/dwyl/mvp/pull/270, so it should be merged after that one :)

### CODE REVIEW

**Comment by LuchoTurtle** [comment]
This should now be reviewable.
Please check https://github.com/dwyl/mvp/pull/270 before merging this one.

File: config/dev.exs [hunk 1]
```diff
@@ -22,7 +22,7 @@ config :app, AppWeb.Endpoint,
   http: [ip: {127, 0, 0, 1}, port: 4000],
   check_origin: false,
   code_reloader: true,
-  debug_errors: true,
+  debug_errors: false,
```
Comment (nelsonic) [inline config/dev.exs:25]: debugging in `dev` is actually desirable ... (IMO) 💭

Comment (LuchoTurtle) [reply config/dev.exs:25]: I agree, hence I was [reluctant to toggle it to `false`](https://github.com/dwyl/mvp/issues/269#issuecomment-1384336847).
However, in order for ErrorView to display `JSON` responses correctly, as it was requested in https://github.com/dwyl/mvp/issues/269, this config needs to be toggled off (as per docs -> https://hexdocs.pm/phoenix/custom_error_pages.html#the-errorview)

Comment (LuchoTurtle) [reply config/dev.exs:25]: https://github.com/dwyl/mvp/pull/270/files#diff-4f36118d05999cec87dec4895c8202881ba0577dc80fba4050aa50bbe9e326b3 (that was just merged) also had this change.

**Review by nelsonic — APPROVED** [review]
Thanks @LuchoTurtle 👌

**Pull request merged**